mizar/dp/mizar/workflows/builtins/services/create.py

```diff
@@ -38,11 +43,19 @@ def requires(self):
 
     def run(self):
         logger.info("Run {task}".format(task=self.__class__.__name__))
-        endpoints_opr.create_scaled_endpoint(
-            self.param.name, self.param.spec, self.param.body['metadata']['namespace'])
+        net = nets_opr.store.get_net(OBJ_DEFAULTS.default_ep_net)
+        if 'arktos.futurewei.com/network' in self.param.body['metadata'].get('labels', {}):
+            arktosnet = self.param.body['metadata']['labels']['arktos.futurewei.com/network']
+            vpc = vpcs_opr.store.get_vpc_in_arktosnet(arktosnet)
+            nets = nets_opr.store.get_nets_in_vpc(vpc)
+            if nets:
+                net = next(iter(nets.values()))
+        ep = endpoints_opr.create_scaled_endpoint(
+                    self.param.name, self.param.spec, net, self.param.body['metadata']['namespace'])
+        if 'clusterIP' not in self.param.spec:
+            kube_patch_service(client.CoreV1Api(), self.param.name, {"spec": {"clusterIP": ep.ip}}, self.param.body['metadata']['namespace'])
```

Comment:
We should remove this as it's Arktos related and needs to be done by the network controller.